src/fides/data/location/locations.yml

```diff
@@ -359,16 +359,19 @@ locations:
   name: Ireland
   belongs_to: [eea]
   regulation: [gdpr]
+  default_selected: True
```

Comment:
this field will support some new OOB location "seeding" that'll be done over in plus